host/tests/L3_TestCases/dsFPD/dsFPDHelperClass.py

```diff
@@ -95,10 +97,6 @@ def testRunPrerequisites(self):
         
         test = self.testSetup.get("assets").get("device").get(self.testName)
```

Comment:
If this function is of no use remove it

Reply:
Removed this function as it is no longer needed for test execution.